OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- arthurdm
+- nebhale
+reviewers:
+- sbose78
+- scothis
```

Comment:
```suggestion
- nebhale
- scothis
reviewers:
- baijum
- sbose78
```

Once Predrag gets the CLA/membership sorted we should add him as an approver as well